atoma-proxy-service/src/proxy_service.rs

```diff
@@ -288,7 +346,7 @@ async fn generate_api_token(
     )
 )]
 #[instrument(level = "info", skip_all)]
-async fn revoke_api_token(
+pub(crate) async fn revoke_api_token(
     State(proxy_service_state): State<ProxyServiceState>,
     headers: HeaderMap,
     body: Json<RevokeApiTokenRequest>,
```

Comment:
This apply to all endpoints. In utoipa docs for this endpoint:
```
(status = OK, description = "Revokes an API token for the user", body = Value),
```
`body = Value`, instead of `body = RevokeApiTokenRequest`

Is it on porpose?

Reply:
Addressed